### DISCUSSION TITLE
Can I read results using the JS Library?

### DISCUSSION THREAD

**Discussion by Ra-Na** [dlubal-software/Dlubal_JavaScript_Library#56] [Q&A]
I would really appreciate a minimum working example for extracting results by using the JS library.

**Comment by jarabroz** [comment]
Hi @Ra-Na,

We are not yet able to extract results in the JS Library. We are using JS for in-application scripting. Main usage scenario is for generating models so that you can create your own Block or to modify existing model in application - [see this KB article](https://www.dlubal.com/en-US/support-and-learning/support/knowledge-base/001689) or use your script in Script manager.

**Comment by Ra-Na** [comment]
I now use a macro recorder for recurring tasks, but its slow and the computer is unusable in the meantime.
